### PR TITLE
feat: implement basic browser

### PR DESCRIPTION
**Pull request by edgarkhanzadian**
https://github.com/leather-io/mono/assets/22010816/24911d6d-11b5-476c-87e5-b1fcde187428

## Summary by CodeRabbit

- **New Features**
	- Introduced a browser screen with empty and active states for navigating websites.
	- Added a navigation button in the collectibles tab for accessing the new browser screen.

- **Refactor**
	- Simplified the `NavigationHeader` structure for improved readability and maintainability.

- **Dependencies**
	- Added `react-native-webview` version `13.8.6`.

- **Internationalization**
	- Added new message ID for "Cancel".
	- Updated message content for "Bitcoin for the rest of us".

### CODE REVIEW

File: apps/mobile/src/app/wallet/(tabs)/collectibles.tsx [hunk 1]
```diff
@@ -1,9 +1,23 @@
 import { StyleSheet, Text, View } from 'react-native';
 
+import { APP_ROUTES } from '@/constants';
```
Comment (kyranjamie) [inline apps/mobile/src/app/wallet/(tabs)/collectibles.tsx:3]: Unrelated to PR but, to align with extension, can the alias be sans `/`?  So `@constants`?

File: apps/mobile/src/app/wallet/browser.tsx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  return 'https://' + url;
+}
+
+function BrowerInUse({
+  textURL,
+  goToInactiveBrowser,
+}: {
+  textURL: string;
+  goToInactiveBrowser: () => void;
+}) {
```
Comment (kyranjamie) [inline apps/mobile/src/app/wallet/browser.tsx:31]: Could declare as separate interface

File: apps/mobile/src/app/wallet/browser.tsx [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+        ) : null}
+      </Box>
+    </Box>
+  );
+}
+
+enum BrowserType {
+  inactive = 'inactive',
+  active = 'active',
+}
```
Comment (kyranjamie) [inline apps/mobile/src/app/wallet/browser.tsx:188]: could be a string literal 🙃

**Review by kyranjamie — APPROVED** [review]
LGTM. 

Where possible I think it'd be nice to continue using layout like pattern from extension, whereby we separate layout & styles in a separate component from the state & logic

File: apps/mobile/src/app/wallet/browser.tsx [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+      );
+
+    case BrowserType.active:
+      return (
+        <BrowerInUse
+          textURL={textURL}
+          goToInactiveBrowser={() => setBrowserType(BrowserType.inactive)}
+        />
+      );
+  }
```
[the remark this answers is not in the document]
Comment (pete-watters) [reply apps/mobile/src/app/wallet/browser.tsx:211]: @edgarkhanzadian I think this should update

File: apps/mobile/src/app/wallet/browser.tsx [hunk 5]
```diff
@@ -0,0 +1,212 @@
+import { useRef, useState } from 'react';
```
Comment (pete-watters) [inline apps/mobile/src/app/wallet/browser.tsx:1]: Maybe this file could be split up into 3? `BrowerInUse`, `BrowserEmptyState` + `BrowserScreen`?

**Pull request merged**